environment.yml

```diff
@@ -1,7 +1,6 @@
 name: ads_venv
 channels:
   - conda-forge
-  - defaults
```

Comment:
Why'd you delete this line? We explored many different configurations in #452, and including the default channel resulted in the best success in our tests (for now). See the long discussion in that PR

Reply:
So, defaults channels are apparently installed at most places and wasn't absolutely needed. I tried delete both conda-forge and defaults (because they generate noise warning during GitHub Action CI) and it empirically past all tests even with defaults so I figured it wasn't absolutely required. 

I think we can totally leave it in for better compatibility if that is a concern as the current GitHub Actions pass with either options.